Add back defaultEntryPoints setting (for HTTP/TCP) (#9100)
traefik/traefik: docs/content/routing/entrypoints.md

    Full details for how to specify `address` can be found in [net.Listen](https://golang.org/pkg/net/#Listen) (and [net.Dial](https://golang.org/pkg/net/#Dial)) of the doc for go.

### AsDefault

_Optional, Default=false_

> The `AsDefault` option flags the EntryPoint to be in the list of default EntryPoints.

Reviewer: ```suggestion
The `AsDefault` option marks the EntryPoint to be in the list of default EntryPoints.
```

> EntryPoints in this list are used (by default) on HTTP and TCP routers that do not define the [EntryPoints option](./routers/index.md#entrypoints).

Reviewer: ```suggestion
EntryPoints in this list are used (by default) on HTTP and TCP routers that do not define their own [EntryPoints option](./routers/index.md#entrypoints).
```

!!! info "List of default EntryPoints"

    If there is no EntryPoint with the `AsDefault` option set to `true`, 
    then the list of default EntryPoints includes all HTTP/TCP EntryPoints.

>     If at least one EntryPoint have the `AsDefault` option set to `true`,

Reviewer: ```suggestion
    If at least one EntryPoint has the `AsDefault` option set to `true`,
```

    then the list of default EntryPoints includes only EntryPoints that have the `AsDefault` option set to `true`.

    Some built-in EntryPoints are always excluded from the list, namely: `traefik`, `traefikhub-api`, and `traefikhub-tunl`.

!!! warning "Only TCP and HTTP"

    The `AsDefault` option has no effect on UDP EntryPoints.

>     When UDP routers are not defining the [EntryPoints option](./routers/index.md#entrypoints_2),
>     they are attached to all available UDP EntryPoints.

Reviewer: ```suggestion
    When a UDP router does not define the [EntryPoints option](./routers/index.md#entrypoints_2),
    it is attached to all available UDP EntryPoints.
```

    Full details for how to specify `address` can be found in [net.Listen](https://golang.org/pkg/net/#Listen) (and [net.Dial](https://golang.org/pkg/net/#Dial)) of the doc for go.

### DefaultSet

_Optional, Default=false_

> The `DefaultSet` option defines whether the EntryPoint should be included in the EntryPoints default set.

Reviewer 2: ```suggestion
The `DefaultSet` option flags the EntryPoint to be used by default when configuring HTTP and TCP routers that do not define the [EntryPoints option](./routers/index.md#entrypoints).
```

> The EntryPoints default set is used to configure HTTP and TCP routers which are not defining the [EntryPoints option](./routers/index.md#entrypoints).

Reviewer 2: ```suggestion
```

!!! info "How the EntryPoints default set is built"

    If no EntryPoint is explicitly included in the EntryPoints default set, 

>     meaning that all EntryPoints have the `DefaultSet` option set to `false`,
>     the EntryPoints default set includes all HTTP/TCP EntryPoints.

Reviewer 2: ```suggestion
    then it includes all HTTP/TCP EntryPoints.
```

    If at least one EntryPoint is explicitly included in the EntryPoints default set,

>     meaning that at least one EntryPoint have the `DefaultSet` option set to `true`,
>     the EntryPoints default set includes only EntryPoints that have the `DefaultSet` option set to `true`.

Reviewer 2: ```suggestion
    then only EntryPoints that have the `DefaultSet` option set to `true` are included.
```
